src/hooks/useUniversalRouter.ts

```diff
@@ -49,6 +49,7 @@ export function useUniversalRouterSwapCallback(
   options: SwapOptions
 ) {
   const { account, chainId, provider } = useWeb3React()
+  const analyticsContext = useTrace()
```

Comment:
seems like its called "trace" throughout the interface codebase (and also in src/pages/Swap/index.tsx where you just added). could we rename it to trace here to keep consistent?

Reply:
i used a different name here to disambiguate from the `trace` function being used to do perf monitoring

Reply:
I'd love for a followup to change the export in `@uniswap/analytics` to just be `useAnalyticsContext` so that it can cleanly be named `analyticsContext`. Could you ticket that and follow up on it?

If we _always_ want the trace to be included with an analytics event, it may also be useful to have a `useSendAnalyticsEvent` hook exported which includes the trace in the sent event properties.

Reply:
https://linear.app/uniswap/issue/WEB-17/analytics-improvements

Reply:
maybe naming here: analyticsTrace?